src/components/Cluster/ClusterDetail/NodePool.tsx

```diff
@@ -260,11 +254,13 @@ class NodePool extends Component<INodePoolsProps, INodePoolsState> {
           style={{ gridColumn: isNameBeingEdited ? '2 / 9' : undefined }}
         >
           <NPViewAndEditNameStyled
-            name={nodePool.name}
-            type='node pool'
-            onSubmit={this.editNodePoolName}
-            ref={(viewEditName: INPViewAndEditName): void => {
-              this.viewEditNameRef = viewEditName;
+            value={nodePool.name}
+            typeLabel='node pool'
+            onSave={this.editNodePoolName}
+            ref={(viewEditName: HTMLSpanElement | null): void => {
+              if (viewEditName) {
+                this.viewEditNameRef = viewEditName as INPViewAndEditName;
+              }
```

Comment:
Why do we need the 'if' statement?

Reply:
In development I changed the type of the `viewEditNameRef` to be not `| null` so I forgot to revert this.

Reply:
I bet it's to make typescript compile, since viewEditName can be null?